serial_linux.go

```diff
@@ -103,6 +104,17 @@ func (p *Port) Write(b []byte) (n int, err error) {
 	return p.f.Write(b)
 }
 
+func (p *Port) Flush() error {
+	const TCFLSH = 0x540B
+	_, _, err := syscall.Syscall(
+		syscall.SYS_IOCTL,
+		uintptr(p.fd),
```

Comment:
Why not just use p.f.Fd() here?  Right now it seems like Port would have redundant information with both File and fd.